spacemacs/packages.el

```diff
@@ -3708,10 +3708,15 @@ one of `l' or `r'."
               (,(listify-key-sequence
                  (kbd dotspacemacs-emacs-leader-key)) . "Spacemacs root")))
       (nconc which-key-prefix-title-alist spacemacs/prefix-titles)
+      (pcase dotspacemacs-which-key-position
+        ('right (which-key-setup-side-window-right))
+        ('bottom (which-key-setup-side-window-bottom))
+        ('right-then-bottom (which-key-setup-side-window-right-bottom)))
```

Comment:
The pcase documentation also explicitly mentions backquotes (backticks? What are they called?).

Also the argument
```elisp
(eq `right 'right) => t
```
is valid only in non-lisps where arguments are always evaluated. However, lisp (and elisp) macros can distinguish between backquote and single quote and behave differently.